### PR TITLE
Semantics in the docs updated

### PR DESCRIPTION
**Pull request by vk-98**

### CODE REVIEW

**Comment by vk-98** [comment]
@rda3mon Please review this change :).

**Comment by rda3mon** [comment]
@vk-98 Looking into it. I somehow missed this.

File: docs/setup/additional/rackawareness.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 1. Build rack utils docker image as below
-    ```
+
+    ```sh
     docker build utilities/rackuitls/ --network host --build-arg AppName="RackUtils" -t "hbase-rack-utils:1.0.0"
     docker push hbase-rack-utils:1.0.0
     ```
 
-1. Enable sidecar along with hmaster container as described below in values.yaml file
+2. Enable sidecar along with hmaster container as described below in values.yaml file
```
Comment (rda3mon) [inline docs/setup/additional/rackawareness.md:12]: If you retain it to be 1. it will auto increment the integer.

**Review by rda3mon — CHANGES_REQUESTED** [review]

File: docs/setup/additional/rackawareness.md (hunk 1 above)
Comment (vk-98) [reply docs/setup/additional/rackawareness.md]: Ohh yeah, reverted back these changes.

**Review by rda3mon — APPROVED** [review]

**Pull request merged**